address.go

```diff
@@ -44,6 +48,39 @@ func encodeAddress(hash160 []byte, netID byte) string {
 	return base58.CheckEncode(hash160[:ripemd160.Size], netID)
 }
 
+// encodeSegWitAddress creates a bech32 encoded address string representation
+// from witness version and witness program.
+func encodeSegWitAddress(hrp string, witnessVersion byte,
+	witnessProgram []byte) (string, error) {
```

Comment:
Please keep function definitions on the same line to be more consistent with the rest of code base.